### DISCUSSION TITLE
Can I publish the article on this platform?

### DISCUSSION THREAD

**Discussion by Allana589** [community/community#58312] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I wanna upload article related to[ can dogs eat cookies?](<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Ayesha/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Ayesha/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

https://petarenapro.com/can-dogs-eat-cookies/
--

</body>

</html>
) What should I do?

**Comment by aahouzi** [comment]
I think u need to find the right place in Github Communities, that's the best answer I can give you :)

**Comment by ghostinhershell** [comment]
Hi @Allana589, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
